build.gradle

```diff
@@ -11,25 +11,19 @@ architectury {
 subprojects {
     apply plugin: "dev.architectury.loom"
 
+    repositories {
+        maven { url = "https://pkgs.dev.azure.com/djtheredstoner/DevAuth/_packaging/public/maven/v1" }
```

Comment:
Can you just make this not a single line? (to match how the file looks like)

Reply:
Followed formatting for repositories, also see: https://github.com/Wynntils/Artemis/blob/e96523ce91cd04c662bc6c6f21d7d2f04db002c1/build.gradle#L57